source/_components/light.limitlessled.markdown

```diff
@@ -80,3 +80,129 @@ Refer to the [light]({{site_root}}/components/light/) documentation for general
+
+### {% linkable_title Example Setup and Configuration %}
+
+## General Overview
+
+1. Install the MiLight phone app
+1. Connect the bridge to your network/phone
+1. Link bulbs to one of 4 groups on phone app
+1. Add config to HomeAssistant's configuration.yaml file
+
+### Install MiLight app
+
+Install from your store of choice. I used v3.7 from the Google Play store.
+
+### Connect the v6 bridge to your phone/network
+
+1. connect phone to home WiFi
+2. open MiLight app
+3. click + to go to configuration
+4. smart link
+5. enter home WiFi ssid and password
+6. press reset button once on bottom briefly, the two lights will start flashing
+7. press Start Configuration button on app
+7. it should say 'Success'
+8. lights will go solid then one will slow flashing
+9. press back on the app 
+10. press 'Searching for device' at bottom of screen
+11. your bridge should now be visible in the list, click on it to get into the configuration
+
+If you want to get to the webmin console for your bridge, use 'Advanced IP Scanner' program or similar network scanner to locate the IP address. username: admin, password: admin. It's a good idea to go in in there and set a static IP address for your bridge.
+
+### Link/bind a bulb
+
+Before you can use HomeAssistant to control your bulbs you have to pair them to the phone app. This sends the config to the bridge. Once linked you don't have to use the MiLight app at all if you don't want to.
+
+Pair your bulbs to the relevant 4 zones (tabs) along the top. These are the four groups that map to the 'numbers' in the configuration.yaml. For example the first tab or zone in the App is '- number: 1' in configuration.yaml.
+
+Make sure you choose the relevant logical remote control in the app for your lights. I messed around for hours and hours trying to use the default remote that had colour and colour temp. 
+
+1. once the bridge is connected to your phone and network
+1. open MiLight 3.7
+1. click on the MiLight device
+1. you will see the 4 zones along the top
+1. click the two arrows icon in the top right to choose the relevant remote type for your bulb 
+1. back on the main page, click the chainlink icon in top right corner to get to the Link/Unlink page
+1. unlink the bulb first (if you have already used it before)
+   1. turn the light you want to connect to off and back on again at the wall
+   1. within 3 secs press the red Unlink button
+   1. light will flash 9 times and will now be unlinked
+1. select the tab/zone along the top for the zone you want to put the bulb into
+1. turn the light you want to connect to off and back on again at the wall
+1. within 3 secs press the green Link button
+1. light will flash 4? times and will now be linked
+1. go back to the main page and you can control it.
+1. Repeat the linking process for each bulb you want connected to the same group, ie. 4x kitchen lights all connected to the same tab.
+
+I Have:
```

Comment:
**Concern**:
Not sure why your collection of hardware matters here. I would suggest removing this section.
_Note: Skipped spelling & grammar review of this section because of that`.